lib/futbol_data.rb

```diff
@@ -0,0 +1,62 @@
+require "csv"
+class FutbolData
+
+  attr_accessor :games, :teams, :game_teams
+
+  def initialize(passed)
+    @passed = passed
+    @data_location = nil
+    create_objects
+  end
+
+  def create_objects
+    chosen_data_set # returns correct @data_location
+    if @passed == 'teams'
+      case_is_team
+    elsif @passed == 'games'
+      case_is_game
+    elsif @passed == 'game_teams'
+      case_is_game_teams
+    end
+  end
+
+  def chosen_data_set
+    case @passed
+      when "teams"
+        @data_location = './data/teams.csv'
+      when "games"
+        @data_location = './data/games.csv'
+      when "game_teams"
+        @data_location = './data/game_teams.csv'
+    end
+    @data_location
+  end
```

Comment:
Great job using cases

Reply:
This was a fun learn! I'm glad it works the way we wanted it to!